beta/src/pages/apis/react/useCallback.md

```diff
@@ -2,17 +2,278 @@
 title: useCallback
 ---
 
-<Wip>
+<Intro>
+
+The `useCallback` hook returns a memoized version of the [callback function](https://en.wikipedia.org/wiki/Callback_(computer_programming)) that is passed inline along with an array of dependencies. When a function is wrapped in `useCallback`, that function is executed only when the dependencies change. This is useful when passing a [callback](https://developer.mozilla.org/en-US/docs/Glossary/Callback_function) to optimized child components that rely on **referential equality to** [**prevent unnecessary renders**](/apis/react/useContext#optimizing-re-renders-when-passing-objects-and-functions).
+
+```js
+const memoizedCallback = useCallback(callback, [...dependencies])
+```
 
-This section is incomplete, please see the old docs for [useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback).
+</Intro>
 
-</Wip>
+<Note>
```

Comment:
Is this worth adding to the `useMemo` docs too?

Reply:
Will add.